lib/stdlib/doc/src/shell.xml

```diff
@@ -282,6 +282,55 @@
           To read all record definitions, use
           <c>'_'</c> as value of <c>RecordNames</c>.</p>
       </item>
+      <tag><c>lf()</c></tag>
+      <item>
+        <p>Outputs locally defined function with function specs if they exist.
+        </p>
+      </item>
+      <tag><c>lt()</c></tag>
+      <item>
+        <p>Outputs locally defined types.
+        </p>
+      </item>
+      <tag><c>lr()</c></tag>
+      <item>
+        <p>Outputs locally defined records.
+        </p>
+      </item>
+      <tag><c>ff()</c></tag>
+      <item>
+        <p>Forget locally defined functions (including function specs if they exist).
+        </p>
+      </item>
+      <tag><c>ff({FunName,Arity})</c></tag>
```

Comment:
I would have expected this to take two arguments, FuncName and Arity, not a two tuple.